assets/src/edit-story/elements/video/controls.js

```diff
@@ -0,0 +1,252 @@
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useCallback, useEffect, useState } from 'react';
+import { useDebouncedCallback } from 'use-debounce';
+import { CSSTransition } from 'react-transition-group';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as PlayIcon } from '../../icons/play.svg';
+import { ReactComponent as PauseIcon } from '../../icons/pause.svg';
+import { useStory } from '../../app/story';
+import StoryPropTypes from '../../types';
+
+const PLAY_BUTTON_SIZE = 48;
+const PLAY_ABOVE_BREAKPOINT_WIDTH = 108;
+const PLAY_ABOVE_BREAKPOINT_HEIGHT = 120;
+
+const Controls = styled.div`
+  position: absolute;
+  z-index: 2;
+  left: ${({ x }) => `${x}px`};
+  top: ${({ y }) => `${y}px`};
+  width: ${({ width }) => `${width}px`};
+  height: ${({ height }) => `${height}px`};
+`;
+
+const ButtonWrapper = styled.div.attrs({ role: 'button', tabIndex: -1 })`
+  position: absolute;
+  top: ${({ isAbove }) => (isAbove ? `-52px` : `50%`)};
+  left: 50%;
+  transform: translate(-50%, -50%);
+  cursor: pointer;
+  pointer-events: initial;
+  width: ${PLAY_BUTTON_SIZE}px;
+  height: ${PLAY_BUTTON_SIZE}px;
+
+  opacity: ${({ isAbove }) => (isAbove ? 1 : 0)};
+  &.button-enter {
+    opacity: 0;
+  }
+  &.button-enter-active,
+  &.button-enter-done {
+    opacity: 1;
+    transition: opacity 100ms;
+  }
+  &.button-exit {
+    opacity: 1;
+  }
+  &.button-exit-active,
+  &.button-exit-done {
+    opacity: 0;
+    transition: opacity 100ms;
+  }
+`;
+
+const Play = styled(PlayIcon)`
+  width: 100%;
+  height: 100%;
+  opacity: 0.84;
+  filter: drop-shadow(0px 0px 10px rgba(0, 0, 0, 0.2));
+`;
+const Pause = styled(PauseIcon)`
+  width: 100%;
+  height: 100%;
+  opacity: 0.84;
+  filter: drop-shadow(0px 0px 10px rgba(0, 0, 0, 0.2));
+`;
+
+function VideoControls({
+  box,
+  isSelected,
+  isSingleElement,
+  isEditing,
+  isTransforming,
+  elementRef,
+  element,
+}) {
+  const isPlayAbove =
+    element.width < PLAY_ABOVE_BREAKPOINT_WIDTH ||
+    element.height < PLAY_ABOVE_BREAKPOINT_HEIGHT;
+  const [hovering, setHovering] = useState(false);
+  const [showControls, setShowControls] = useState(true);
+  const [isPlaying, setIsPlaying] = useState(!isTransforming);
+  const { id } = element;
+  const {
+    state: { selectedElementIds },
+  } = useStory();
+  const isElementSelected = selectedElementIds.includes(id);
```

Comment:
Isn't `isSelected` the same thing already?

Reply:
Yep, done in https://github.com/google/web-stories-wp/pull/1948/commits/8e319c603411f7ac0759bb91884493388e4fd84b